Formula/c/cucumber-cpp.rb

```diff
@@ -16,30 +17,32 @@ class CucumberCpp < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "nlohmann-json" => :build
+  # TODO: qt6 discovery not working, it finds 6.6.x but says it's unsuitable
+  depends_on "qt@5" => :build
   depends_on "ruby" => :test
-  depends_on "boost"
+  depends_on "asio"
+  depends_on "tclap"
 
   def install
     args = %w[
       -DCUKE_DISABLE_GTEST=on
-      -DCUKE_DISABLE_CPPSPEC=on
-      -DCUKE_DISABLE_FUNCTIONAL=on
-      -DCUKE_DISABLE_BOOST_TEST=on
-      -DCMAKE_CXX_STANDARD=17
+      -DCUKE_ENABLE_EXAMPLES=on
     ]
 
     system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
     system "cmake", "--build", "build"
     system "cmake", "--install", "build"
+
+    doc.install "examples"
```

Comment:
Ideally I'd like a way to also run at least 1-2 of these examples during `test`, it would also help verify whether the `Qt` samples work